docs/alerts.md

```diff
@@ -91,7 +245,41 @@ The elasticsearch Cluster is predicted to be out of disk space within the next 6
 
 ### Troubleshooting
 
-Check the disk space of the Elasticsearch node.
+1. Check the disk space of the Elasticsearch node:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_nodes/stats/fs?pretty
+   ```
+   Check the “nodes.node_name.fs” field to determine the free disk space on that node.
+
+2. The prediction is done based on the current trend in occupying the disk space on that node.
+
+3. If it looks like the disk space will not get over then
+    - Ignore the alert.
+    
+4. If it looks like the disk space will get over soon then
+    - Increase disk space on all nodes.
+    - If increasing disk space isn’t possible, then add a new data node to the cluster.
+    - If adding a new data node is problematic, then reduce the number of replicas
+        - Preferably on older indices.
+        - Get the index settings:
+          ```
+          oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty
+          ```
+        - Identify the number of replicas from the output of the above command.
```

Comment:
the problem here is that the elasticsearch operator would revert this. so we would probably need to have them reduce their total cluster redundancy policy. e.g. If they are using something higher than `SingleRedundancy` reduce it to `SingleRedundancy` (we can't tell them to use zero replicas since that is dangerous from a data loss perspective)

Reply:
Added that step above "reduce the number of replicas" and changed the statement to reduce replicas not less than 1.